### PR TITLE
Add random/1.3.1

### PR DESCRIPTION
**Pull request by prince-chrismc**
Specify library name and version:  **random/1.3.1**

closes #1354

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/effolkronium_random/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+
+from conans import ConanFile, tools
+
+class RandomConan(ConanFile):
+    name = "effolkronium_random"
```
Comment (uilianries) [inline recipes/effolkronium_random/all/conanfile.py:6]:
```suggestion
    name = "effolkronium-random"
```

Use `-`, not `_`. There are other projects: bvdberg-ctest, kainjow-mustache, llvm-openmp, mpark-variant, taocpp-json ...

File: recipes/effolkronium_random/all/test_package/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.11)
```
Comment (uilianries) [inline recipes/effolkronium_random/all/test_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

https://github.com/conan-io/conan-center-index/issues/2184

File: recipes/effolkronium_random/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/effolkronium_random/all/conanfile.py:6]: `_` is from the cmake lists... On mobile I can grab the link after

Comment (uilianries) [reply recipes/effolkronium_random/all/conanfile.py:6]: Yeah, but we already have this discussion for mpark-variant, take a look https://github.com/conan-io/conan-center-index/pull/1269

Comment (prince-chrismc) [reply recipes/effolkronium_random/all/conanfile.py:6]: Ahhh okay... The cmake targets need to be configured for that project then.

Would it be possible to have https://github.com/conan-io/conan-center-index/wiki/FAQ#should-reference-names-use---or-_ update to be more clear on the topic? 

> Recipes should stick to the original name of a library as much as possible 

"however sticking with <author>-<project> is the preferred manner"

In this case, everything is written `_` but [vcpkg](https://github.com/microsoft/vcpkg/search?q=effolkronium&unscoped_q=effolkronium) is `-`

Comment (uilianries) [reply recipes/effolkronium_random/all/conanfile.py]: Indeed it can be. We should move wiki to docs folder soon, so anyone will be able to PR :)

**Pull request merged**